cloudhub-2/modules/ROOT/pages/ch2-understand-app-restart.adoc

```diff
@@ -0,0 +1,35 @@
+= Understanding App Automatic Restart 
+
+
+In some scenarios, the platform can restart your Mule application. 
```

Comment:
This is the short description - can you reword it to summarize the topic?

The CloudHub 2.0 platform automatically restarts your Mule application when xyz..